tests/pr_test.go

```diff
@@ -95,6 +96,15 @@ func TestRunFullyConfigurableSchematics(t *testing.T) {
 		{Name: "region", Value: validRegions[rand.Intn(len(validRegions))], DataType: "string"},
 		{Name: "existing_resource_group_name", Value: resourceGroup, DataType: "string"},
 		{Name: "service_plan", Value: "trial", DataType: "string"},
+		{Name: "secrets", Value: []map[string]interface{}{
```

Comment:
you have duplicated this code in every test. Suggest to store it as a global variable and re-use it instead of duplicating it